src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-5-0-317.mdx

```diff
@@ -0,0 +1,27 @@
+---
+subject: PHP agent
+releaseDate: '2023-01-17'
+version: 10.5.0.317
+downloadLink: 'https://download.newrelic.com/php_agent/archive/10.5.0.317'
+---
+## New Relic PHP Agent v10.5.0.317
+
+### Bug Fixes
+* agent: Fixed [#577](https://github.com/newrelic/newrelic-php-agent/issues/577) - Large performance regression in 10.3.0. [#588](https://github.com/newrelic/newrelic-php-agent/pull/588)
+* agent: Fixed [#594](https://github.com/newrelic/newrelic-php-agent/issues/594) - Instrumentation from `opcache.preload` for PHP 8.1. [#595](https://github.com/new^relic/newrelic-php-agent/pull/595)
+* agent: Fixed sending `supportability` metrics for application logging configuration. [#575](https://github.com/newrelic/newrelic-php-agent/pull/575)
+
+### Support statement
+* Support for 32 bit and FreeBSD ended with release v9.19.0.309 and those binaries **are no longer provided**.
+* Support for ZTS builds ended with release v9.19.0.309.
+* New Relic recommends that you upgrade the agent regularly and at a minimum every 3 months. For more information on supported agent versions and EOL timelines, check out our [PHP EOL policy](https://docs.newrelic.com/docs/apm/agents/php-agent/getting-started/php-agent-eol-policy/).
+* The [PHP agent compatibility and requirements](https://docs.newrelic.com/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements/) page should be consulted for the latest information on supported PHP versions and platforms.
+
+<Callout variant="important">
+**For installations using an unsupported PHP version or platform (32 bit, FreeBSD, ZTS, etc), it's highly recommended that you disable the auto-update mechanisms for the PHP agent packages.** This can be done by adding the PHP agent packages to an exclusion list for package upgrades. Or you could version pin the PHP agent package to an agent version that supports the old, unsupported feature(s). Failure to prevent upgrades may result in a newer agent release being installed and remove support for the required, unsupported features. This would disrupt APM data collection.  
```

Comment:
```suggestion
**For installations using an unsupported PHP version or platform (32 bit, FreeBSD, ZTS, etc), it's highly recommended that you disable the auto-update mechanisms for the PHP agent packages.** This can be done by adding the PHP agent packages to an exclusion list for package upgrades. Or you could version pin the PHP agent package to an agent version that supports the old, unsupported feature(s). Failure to prevent upgrades may result in a newer agent release being installed and removal of support for the required, unsupported features. This would disrupt APM data collection.
```